src/main/java/school/lemon/changerequest/java/introduction/hw1/Task10.java

```diff
@@ -4,6 +4,12 @@
 public class Task10 {
 
     public static boolean containDigitTwo(int n) {
+        while(n >= 2)
```

Comment:
And what if `n` will be equal to `2` ?
Your solution will return `false` instead of `true`.

Please, think about better implementation.